lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -851,8 +852,12 @@ def setUp(self):
         self.student = UserFactory.create()
         CourseEnrollmentFactory(user=self.student, course_id=self.course.id)
 
+    @patch('lms.lib.comment_client.comment.Comment.thread')
```

Comment:
is it necessary to patch this module even though it's not being referenced during the method?